webpack.common.js

```diff
@@ -64,7 +64,9 @@ module.exports = {
       SAVE_DEBOUNCE_TIME: JSON.stringify(process.env.SAVE_DEBOUNCE_TIME),
       SEARCH_DEBOUNCE_TIME: JSON.stringify(process.env.SEARCH_DEBOUNCE_TIME),
       SUBSCRIPTION_SERVICE: JSON.stringify(process.env.SUBSCRIPTION_SERVICE
-        || 'http://localhost:3000'),
+        || 'https://alpha-payment.rovercode.com'),
```

Comment:
I would prefer this to be the local service for development

Reply:
Didn't mean to commit that change. Putting it back.